types/react-sortable-hoc/index.d.ts

```diff
@@ -34,21 +34,105 @@ declare module 'react-sortable-hoc' {
+         * Cannot be used in conjunction with the `pressDelay` prop.
+         */
+        distance?: number;
+        
+        /**
+         * This function get's invoked before sorting begins, and can be used to programatically cancel
+         * sorting before it begins. By default, it will cancel sorting if the event target is either an
+         * `input`, `textarea`, `select` or `option`.
+         */
+        shouldCancelStart?: (event: SortEvent) => boolean;
+        
+        /**
+         * Callback that get's invoked when sorting begins. `function({node, index, collection}, event)`
+         */
         onSortStart?: SortStartHandler;
+        
+        /**
+         * Callback that get's invoked during sorting as the cursor moves. `function(event)`
+         */
         onSortMove?: SortMoveHandler;
+        
+        /**
+         * Callback that get's invoked when sorting ends. `function({oldIndex, newIndex, collection}, e)`
+         */
         onSortEnd?: SortEndHandler;
+        
+        /**
+         * If you're using the `SortableHandle` HOC, set this to `true`
+         */
         useDragHandle?: boolean;
+        
+        /**
+         * If you want, you can set the `window` as the scrolling container
+         */
         useWindowAsScrollContainer?: boolean;
+        
+        /**
+         * Whether to auto-hide the ghost element. By default, as a convenience, React Sortable List will
+         * automatically hide the element that is currently being sorted. Set this to false if you would
+         * like to apply your own styling.
+         */
         hideSortableGhost?: boolean;
+        
+        /**
+         * You can lock movement of the sortable element to it's parent `SortableContainer`
+         */
         lockToContainerEdges?: boolean;
+        
+        /**
+         * `"50%"`
+         */
         lockOffset?: Offset | [Offset, Offset];
+        
+        /**
+         * Optional function to return the scrollable container element. This property defaults to the
+         * `SortableContainer` element itself or (if `useWindowAsScrollContainer` is true) the window. Use
+         * this function to specify a custom container object (eg this is useful for integrating with
+         * certain 3rd party components such as `FlexTable`). This function is passed a single parameter
+         * (the `wrappedInstance` React element) and it is expected to return a DOM element.
+         */
         getContainer?: ContainerGetter;
-        shouldCancelStart?: (event: SortEvent) => boolean;
+        
+        /**
+         * Optional `function({node, index, collection})` that should return the computed dimensions of the
+         * SortableHelper. See [default implementation][0] for more details
+         * [0]: https://github.com/clauderic/react-sortable-hoc/blob/v0.4.5/src/SortableContainer/index.js#L47
+         */
+        getHelperDimensions?: any;
```

Comment:
@Kovensky 

it seems like the "default implementation" doesn't use all the possible arguments as documented: `function({node, index, collection})`. I wasn't too sure on the type of the function which is why I gave it `any`.